src/Tribe/iCal.php

```diff
@@ -304,7 +317,7 @@ private function get_month_view_events() {
 			'hide_upcoming' => true,
 		];
 
-		// Verify the Intial Category
+		// Verify the Initial Category
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 3c8b537c554ea2efe97adb05343f701d</sub>